exporter/elasticsearchexporter/internal/serializer/otelserializer/serializeprofiles/model.go

```diff
@@ -130,3 +133,23 @@ func NewExeMetadata(docID string, lastSeen uint32, buildID, fileName string) Exe
 		},
 	}
 }
+
+// ExecutableSymbolizationData represents an array of executable FileIDs written into the
+// executable symbolization queue index.
+type ExecutableSymbolizationData struct {
+	EcsVersion
```

Comment:
There is a riddle to solve (nothing that disturbs functionality of this PR): The EcsVersion is JSON marshalled as empty string here and for `LeafFrameSymbolizationData` below. But for other types (further up in the file), these are correctly marshalled to "1.12.0".

Reply:
Oh, nvm. Just forgot to explicitly initialize it 😆